### PR TITLE
filename and coresponding changes done for pages/*

### PR DESCRIPTION
**Pull request by bansalshashank**
#2515

### CODE REVIEW

File: build/scss/parts/_pages.scss [hunk 1]
```diff
@@ -8,5 +8,5 @@
 @import '../pages/404_500_errors';
 @import '../pages/invoice';
 @import '../pages/profile';
-@import '../pages/e_commerce';
+@import '../pages/e-commerce';
```
Comment (REJack) [inline build/scss/parts/_pages.scss]: You missed to rename the file too😄

Comment (bansalshashank) [reply build/scss/parts/_pages.scss]: i did rename this file as well its in `pages/examples/e-commerce.html`

Comment (REJack) [reply build/scss/parts/_pages.scss]: But the marked line points to SCSS file not to a HTML :D

Comment (bansalshashank) [reply build/scss/parts/_pages.scss]: change done

**Pull request merged**